tests/unit/spec/base/core-datalayer.js

```diff
@@ -0,0 +1,30 @@
+/* For reference read the Jasmine and Sinon docs
+ * Jasmine docs: http://pivotal.github.io/jasmine/
+ * Sinon docs: http://sinonjs.org/docs/
+ */
+
+/* global describe, beforeEach, afterEach, it, expect, sinon, spyOn, jasmine */
+
+describe('core-datalayer.js', function() {
+
+    describe('PageId', function(){
+        var html = document.documentElement;
+
+        afterEach(function() {
+            html.removeAttribute('data-gtm-page-id');
+        });
+
+        it('will grab data-gtm-page-id value if present on <html> element', function(){
+            html.setAttribute('data-gtm-page-id', 'test');
+
+            expect(Mozilla.Analytics.getPageId()).toBe('test');
```

Comment:
For good measure let's pass in a `path` value here as well, just to make it clear that the data attitude is being used instead.